clang/include/clang/Driver/Options.td

```diff
@@ -6982,8 +6987,8 @@ defm : FlangIgnoredDiagOpt<"target-lifetime">;
 
 // C++ SYCL options
 let Group = sycl_Group in {
-def reuse_exe_EQ : Joined<["-"], "reuse-exe=">, Visibility<[ClangOption, CLOption, DXCOption]>,
-  HelpText<"Speed up FPGA aoc compile if the device code in <exe> is unchanged.">,
+def reuse_exe_EQ : Joined<["-"], "reuse-exe=">,
+  Visibility<[ClangOption, CLOption, DXCOption]>, Flags<[UnsupportedRemoved]>,
```

Comment:
If this option is visible in DXC mode, are those tests removed as well?

Reply:
AFAIK, there are no DXC specific tests for `-reuse-exe`.